app/components/services/services.html

```diff
@@ -45,6 +45,12 @@
                 </a>
               </th>
               <th>
+                <a ui-sref="services" ng-click="order('Status')">
+                  Status
+                  <span ng-show="sortType == 'Status' && !sortReverse" class="glyphicon glyphicon-chevron-down"></span>
+                  <span ng-show="sortType == 'Status' && sortReverse" class="glyphicon glyphicon-chevron-up"></span>
```

Comment:
You can use [ngClass](https://docs.angularjs.org/api/ng/directive/ngClass) to assign a corresponding class:

```html
<span ng-show="sortType === 'Status'" class="glyphicon" ng-class="{
    'glyphicon-chevron-down': !sortReverse,
    'glyphicon-chevron-up': sortReverse
}">
```